src/ui/Facet/FacetSearchElement.ts

```diff
@@ -7,25 +7,27 @@ import { EventsUtils } from '../../utils/EventsUtils';
 import { PopupUtils, PopupHorizontalAlignment, PopupVerticalAlignment } from '../../utils/PopupUtils';
```

Comment:
I inspected the element while NVDA is active as you asked JF. Here is what I hear when I click enter on the facet search div:

```
Combobox expanded. Search edit has autocomplete. Blank. List.

Select task with 140 results. 140 Task exclude Task. 1 of 15.
```

When using the arrow keys to navigate, only the second line is read.

```
Select <facet value> with <count> results. <count> <facet value> exclude <facet value>. 1 of 15.
```

Is that similar to what you hear on Mac?

Reply:
Yep that's sound right! Thanks a lot Sami!